[WIP] Argo CD getting started guide. (#30615)
openshift/openshift-docs: cicd/gitops/configuring-a-cluster-with-red-hat-openshift-gitops.adoc

[id="configuring-a-cluster-with-red-hat-openshift-gitops"]

> = Configuring a cluster with Red Hat OpenShift GitOps

Author: That is a very counterintuitive title... But I'll change it.

> = Configuring a cluster with Red Hat OpenShift GitOps

Author: By counterintuitive I mean it describes the process but not the goal. Users won't ever need to *just* sync the repo to the app, the goal of this particular story is to configure your cluster by the means of Argo CD.

include::modules/common-attributes.adoc[]
:context: configuring-a-cluster-with-red-hat-openshift-gitops

toc::[]

With Red Hat OpenShift GitOps, you can define OpenShift cluster configurations such as OpenShift Web Console customizations as well as namespaces by using YAML files and storing them in a Git repository.

> This section describes steps to create link:https://github.com/siamaksade/openshift-gitops-getting-started/tree/main/cluster[sample cluster configurations] which add a link to _Red Hat Developer Blog_ in the Application Launcher and  define a namespace `spring-petclinic` on the cluster.

Reviewer: ```suggestion
You can create link:https://github.com/siamaksade/openshift-gitops-getting-started/tree/main/cluster[sample cluster configurations] that add a link to _Red Hat Developer Blog_ in the Application Launcher and define a namespace `spring-petclinic` on the cluster.
```

Reviewer: IBMSG: Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

Reviewer: IBMSG: Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a nonrestrictive clause.
